packages/dev/parcel-packager-ssg/SSGPackager.js

```diff
@@ -53,8 +53,6 @@ module.exports = new Packager({
       return p.concat(bundles);
     }, []);
 
-    bundles.reverse();
```

Comment:
No longer necessary because of https://github.com/parcel-bundler/parcel/pull/4469